.github/workflows/ci.yml

```diff
@@ -0,0 +1,781 @@
+#############################################################
+#   WARNING: automatically generated file, DO NOT CHANGE!   #
+#############################################################
+
+# This file was automatically generated by the expand-yaml-anchors tool. The
+# source file that generated this one is:
+#
+#   src/ci/github-actions/ci.yml
+#
+# Once you make changes to that file you need to run:
+#
+#   ./x.py run src/tools/expand-yaml-anchors/
+#
+# The CI build will fail if the tool is not run after changes to this file.
+
+---
+name: CI
+"on":
+  push:
+    branches:
+      - auto
+      - try
+      - master
+  pull_request:
+    branches:
+      - "**"
+jobs:
+  pr:
+    name: PR
+    env:
+      CI_JOB_NAME: "${{ matrix.name }}"
+      SCCACHE_BUCKET: rust-lang-gha-caches
+      TOOLSTATE_REPO: "https://github.com/pietroalbini/rust-toolstate"
```

Comment:
I presume we'll be updating these to point at correct locations when we decide to switch over, roughly speaking?

Reply:
Yep.